group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -2575,6 +2576,10 @@ private CoordinatorResult<Map.Entry<ShareGroupHeartbeatResponseData, Optional<In
             records
         ) || initializedAssignmentPending(group);
 
+        if (bumpGroupEpoch) {
+            metrics.record(SHARE_GROUP_REBALANCES_SENSOR_NAME);
```

Comment:
Is there any reason why this metric is not recorded on line 2618, just after the `log.info("[GroupId {}] Bumped group epoch...`?

Reply:
Thanks. I'm just aiming to get the different group types to do this stuff in the same way.